docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1707 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #108.            r_updated = max(r_updated * decay[len(out)], 1e-6)
</pre><p>Add comment for <code>config.STUMPY_MIN_DISCORD_DISTANCE</code> here as well for future reminder </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
I will add it in the next set of commits.